piqueserver/core_commands/movement.py

```diff
@@ -17,49 +17,74 @@ def unstick(connection, player=None):
     player.set_location_safe(player.get_location())
 
 
-@command('goto', admin_only=True)
-def go_to(connection, value):
+@command('moves', admin_only = True)
+def move_silent(connection, *args):
     """
-    Go to a specified sector (e.g. A5) and inform everyone on the server of it
-    /goto <sector>
-    If you're invisivible, it will happen silenty
+    Silently move yourself or of a given player to a specified sector (e.g. A5).
+    If the x/y/z coordinate makes the player appear outside of the world bounds, take the bound instead
+    /moves <sector> [player]
+    /moves <x> <y> <z> [player]
     """
-    if connection not in connection.protocol.players:
-        raise KeyError()
-    move(connection, connection.name, value, silent=connection.invisible)
+    do_move(connection, args, True)
 
 
-@command('gotos', admin_only=True)
-def go_to_silent(connection, value):
+@command(admin_only=True)
+def move(connection, *args):
     """
-    Silently go to a specified sector
-    /gotos <sector>
+    Move yourself or of a given player to a specified sector (e.g. A5) and inform everyone on the server of it.
+    If you're invisivible, it will happen silenty.
+    If the x/y/z coordinate makes the player appear outside of the world bounds, take the bound instead
+    /move <sector> [player]
+    /move <x> <y> <z> [player]
     """
     if connection not in connection.protocol.players:
-        raise KeyError()
-    move(connection, connection.name, value, True)
+            raise ValueError()
+    do_move(connection, args)
+
+
+def do_move(connection, args, silent = False):
+    position = None
+    player = None
+    nbArgs = len(args)
```

Comment:
This project follows the pep8 convention, so we use `snake_case`. As with the indentation thing above, you should probably run something like pylint on your files to check your files for pep8 violations and other style errors.